aviary/interface/methods_for_level2.py

```diff
@@ -685,6 +686,9 @@ def run_aviary_problem(
             with open('input_list.txt', 'w') as outfile:
                 self.model.list_inputs(out_stream=outfile)
 
+        # Creates a flag to determine if the user would or would not like a payload/range diagram
+        payload_range_bool = self.aviary_inputs.get_val(Settings.PAYLOAD_RANGE)
```

Comment:
```suggestion
        payload_range_bool = False
        if Settings.PAYLOAD_RANGE in self.aviary_inputs:
            payload_range_bool = self.aviary_inputs.get_val(Settings.PAYLOAD_RANGE)
```
We don't know that Settings.PAYLOAD_RANGE was always added to the csv, so we need to check it exists first.